### PR TITLE
Update paper.md

### PR DESCRIPTION
**Pull request by arfon**
👋 @jponge this PR updates the JOSS paper to make it compatible with the new submission format.

### CODE REVIEW

File: misc/joss-paper/paper.md [hunk 1]
```diff
@@ -10,21 +10,32 @@ tags:
...
-    affiliation: Blaise Pascal University, LIMOS, F-63170 Aubière, France
+    affiliation: 2
   - name: Frédéric Le Mouël
-    affiliation: Univ Lyon, INSA Lyon, CITI, F-69621 Villeurbanne, France
+    affiliation: 1
   - name: Nicolas Stouls
-    affiliation: Univ Lyon, INSA Lyon, CITI, F-69621 Villeurbanne, France
+    affiliation: 1
   - name: Philippe Charrière
-    affiliation: GitHub, Inc.
```
Comment (arfon) [inline misc/joss-paper/paper.md]: I'm not sure what the correct affiliation for `Philippe Charrière` is here?

File: misc/joss-paper/paper.md [hunk 2]
```diff
@@ -10,21 +10,32 @@ tags:
...
+  - name: Blaise Pascal University, LIMOS, F-63170 Aubière, France
+    index: 2
+  - name: Manufacture Française des Pneumatiques Michelin
+    index: 3
+  - name: Almerys
+    index: 4
+  - name: Omada Health, Inc.
+    index: 5
 date: 23 September 2016
 bibliography: paper.bib
```
Comment (arfon) [inline misc/joss-paper/paper.md]: Please note, you need to actually cite your references in the summary body for them to be included.

**Comment by jponge** [comment]
Philippe is a GitHub employee for real 😄

**Comment by arfon** [comment]
> Philippe is a GitHub employee for real 😄

Lol, sorry!

**Comment by arfon** [comment]
OK, that affiliation should be fixed now.

**Comment by jponge** [comment]
Thanks @arfon for the proposed changes.

We would need that you follow https://github.com/eclipse/golo-lang/blob/master/CONTRIBUTING.md and sign-off your commits with an email address for which we would have [an Eclipse contributor agreement](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git#Eclipse_Contributor_Agreement).

You may just amend / rebase and `git push --force` your branch 😉

**Comment by arfon** [comment]
Could you just make these changes yourself? I don't really want to sign a CLA for this minor fix 😀

**Comment by jponge** [comment]
I'll have to manually do that and sign-off on your behalf then :-)

**Comment by arfon** [comment]
> I'll have to manually do that and sign-off on your behalf then :-)

Sure. Please do whatever you need to. As it stands, the current `paper.md` in here isn't formed correctly so this PR is an attempt to fix the structure. I'd really rather not go through the process of signing up for an Eclipse account and signing a CLA for this structure change.

**Comment by jponge** [comment]
@arfon I'm including your changes with your changes, and ensuring you get the commit authorship with my sign-off.

**Pull request closed without merging**